kube/src/main/java/org/openshift/ping/kube/KubePing.java

```diff
@@ -195,8 +195,8 @@ public void destroy() {
         try {
             pods = client.getPods(_namespace, _labels);
         } catch (Exception e) {
-            if (log.isWarnEnabled()) {
-                log.warn(String.format("Problem getting Pod json from Kubernetes %s for cluster [%s], namespace [%s], labels [%s]; encountered [%s: %s]",
+            if (log.isTraceEnabled()) {
```

Comment:
Instead of changing the log level, is this something that we could use a flag, whereby if it isn't set, we log the message and then set the flag so we don't continue to log the message.  Likewise, we should clear the flag if when we're successful, so that we log the message again if we fail in the future.  This latter part is probably unnecessary, but it could help, for instance if the service account permissions changed while the pod was executing.